Jenkinsfile.kola.aws

```diff
@@ -0,0 +1,59 @@
+def utils, streams
+node {
+    checkout scm
+    utils = load("utils.groovy")
+    streams = load("streams.groovy")
+    pod = readFile(file: "manifests/pod.yaml")
+}
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      string(name: 'BUILDID',
+             description: 'Fedora CoreOS Build ID to test',
+             defaultValue: '',
+             trim: true),
+      string(name: 'S3_STREAM_DIR',
```

Comment:
Because this is a generic job it doesn't have a developer split so it cannot construct the `S3_STREAM_DIR`, as such my intention was to be able to still support running AWS tests in developer pipelines.